atom/renderer/renderer_client_base.cc

```diff
@@ -96,16 +96,14 @@ void RendererClientBase::RenderThreadStarted() {
 #endif
 
 #if defined(OS_MACOSX)
-  // Disable rubber banding by default.
   base::CommandLine* command_line = base::CommandLine::ForCurrentProcess();
-  if (!command_line->HasSwitch(switches::kScrollBounce)) {
-    base::ScopedCFTypeRef<CFStringRef> key(
-        base::SysUTF8ToCFStringRef("NSScrollViewRubberbanding"));
-    base::ScopedCFTypeRef<CFStringRef> value(
-        base::SysUTF8ToCFStringRef("false"));
-    CFPreferencesSetAppValue(key, value, kCFPreferencesCurrentApplication);
-    CFPreferencesAppSynchronize(kCFPreferencesCurrentApplication);
-  }
+  bool scroll_bounce = command_line->HasSwitch(switches::kScrollBounce);
+  base::ScopedCFTypeRef<CFStringRef> rubber_bandking_key(
+    base::SysUTF8ToCFStringRef("NSScrollViewRubberbanding"));
+  CFPreferencesSetAppValue(rubber_bandking_key,
+                           scroll_bounce ? kCFBooleanTrue : kCFBooleanFalse,
+                           kCFPreferencesCurrentApplication);
+  CFPreferencesAppSynchronize(kCFPreferencesCurrentApplication);
```

Comment:
> If no option is passed, use the default value from the operating system

I think this would "break" existing apps that aren't setting the `scrollBounce` key currently since it would use the OS setting and possibly be enabled unknowingly.

Does that sound right to you?

I like the idea of somehow supporting the OS default though 👍 .